src/carbon_calculator/carbonCalculator.py

```diff
@@ -43,20 +46,32 @@ def SavePic2Media(picURL):
             print("ERROR: Unable to import action photo from "+picURL)
             return None
         else:
-            file_name =  "carbon_calculator/assets/" + picURL.split("/")[-1]  # There's probably a better way of doing this but this is just a quick example
-            fp = open(file_name,"wb")
-            fp.write(resp.content)
-            fp.close()
-            media = CarbonCalculatorMedia(file = file_name)
-            #media = CarbonCalculatorMedia.objects.create(file=file_name)
+            image = resp.content
+            file_name =  picURL.split("/")[-1]
+            file_type_ext = file_name.split(".")[-1]
```

Comment:
we should probably make sure that it contains `.` inside first or at least check that `file_type_ext` is not empty before proceeding

Reply:
ok, looks safe to me since inside try, but I added a check